recipes-soc/sysint-soc/sysint-soc.bb

```diff
@@ -0,0 +1,20 @@
+SUMMARY = "Sysint application - Vendor"
+SECTION = "console/utils"
+LICENSE = "CLOSED"
```

Comment:
I think this should be Apache-2.0 please?  (I will comment on the new repo shortly - it is missing the proper licensing files.)